Cargo.toml

```
@@ -9,6 +9,8 @@ edition = "2018"
 
 [features]
 default = ["colored", "chrono"]
+color = ["colored"]
+ts = ["chrono"]
```

```suggestion
timestamps = ["chrono"]
```

No need to optimize for character count here IMO :)